slamd/design_assistant/processing/llm_service.py

```diff
@@ -83,28 +130,28 @@ def _generate_powders_prompt_excerpt(cls, zero_shot_learner_session):
         return powders_prompt_excerpt
 
     @classmethod
-    def _generate_liquids_prompt_excerpt(cls, zero_shot_learner_session):
+    def _generate_liquids_excerpt(cls, zero_shot_learner_session):
         liquid = zero_shot_learner_session['liquid']
         liquid_prompt_excerpt = f'//Liquid : {liquid} \n'
         return liquid_prompt_excerpt
 
     @classmethod
-    def _generate_other_prompt_excerpt(cls, zero_shot_learner_session):
+    def _generate_other_excerpt(cls, zero_shot_learner_session):
         other = zero_shot_learner_session['other']
         other_prompt_excerpt = ''
         if not other == 'None':
             other_prompt_excerpt = f'//Additional components : {other} \n'
         return other_prompt_excerpt
 
     @classmethod
-    def _generate_comment_prompt_excerpt(cls, zero_shot_learner_session):
+    def _generate_comment_excerpt(cls, zero_shot_learner_session):
         comment = zero_shot_learner_session['comment']
         if comment.strip():
             comment_prompt_excerpt = f'//Additional design information : {comment} \n'
         return comment_prompt_excerpt
 
     @classmethod
-    def _generate_design_targets_prompt_excerpt(cls, zero_shot_learner_session):
+    def _generate_design_targets_excerpt(cls, zero_shot_learner_session):
```

Comment:
In den Zeile 158ff -> die choices hat Christoph auf "increase" und "decrease" angepasst (siehe entsprechende Form). Bitte prüfen, ob das konsistent mit der Logik unten ist.